elliptic-curve/Cargo.toml

```
@@ -47,10 +47,10 @@ alloc = ["der/alloc", "sec1/alloc", "zeroize/alloc"] # todo: use weak activation
 arithmetic = ["ff", "group"]
 bits = ["arithmetic", "ff/bits"]
 dev = ["arithmetic", "hex-literal", "pem", "pkcs8"]
+digest = ["digest_traits", "ff", "group"]
```

Perhaps naming this `hash2curve` would be clearer?

```suggestion
hash2curve = ["arithmetic", "digest"]
```

---

What happened to gating behind `digest`?!?

---

See https://github.com/RustCrypto/traits/pull/855#discussion_r777522971 and https://github.com/RustCrypto/traits/pull/863.

---

> What happened to gating behind digest?!?

The changes in this PR:

The `hashing` feature used to have no other effect on other crate dependencies and was therefore redundant.

Now you do want to activate additional features, and to do that you're renaming the `digest` feature.

Since you now need that level of indirection, I'd suggest keeping `digest` as the crate dependency, and adding a separately named (and more specifically named) feature to handle that indirection, rather than renaming the package.

`digest` can now also be independently used for hash-to-scalar without activating `hash2curve`.